src/matchers/toSatisfyAny.js

```diff
@@ -1,23 +1,28 @@
 export function toSatisfyAny(actual, expected) {
   const { printReceived, printExpected, matcherHint } = this.utils;
 
+  const pass = actual.some(expected);
+  const failingElement = pass ? actual.find(expected) : actual.find(a => !expected(a));
```

Comment:
Why return the first non-matching element? In the case of `satisfyAny`, all elements would fail the predicate.